04_IFT/docker/scripts/start.sh

```diff
@@ -13,4 +13,5 @@ set -Eeuoxa pipefail
 LOCAL_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 
 PYTHONPATH="${LOCAL_DIRECTORY}/.." python3 "${LOCAL_DIRECTORY}/../generate_data.py"
+PYTHONPATH="${LOCAL_DIRECTORY}/.." python3 "${LOCAL_DIRECTORY}/../train.py"
```

Comment:
Let's keep data generation and training separate.
Data generation is a long running process on the client. Training is just submitting the generated data. Doing this will break the ./generate_data.sh script

Reply:
Separated the generate_data.sh and train.sh